### PR TITLE
Bump euclid to 0.14

### PR DESCRIPTION
**Pull request by nical**
In order for this to build we need servo-skia to be published on crates.io.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/rust-azure/264)

### CODE REVIEW

**Comment by jdm** [comment]
```
error[E0308]: mismatched types

   --> src/azure_hl.rs:739:71

    |

739 |                                               &AzPoint::from_point_2d(offset),

    |                                                                       ^^^^^^ expected struct `euclid::TypedPoint2D`, found struct `euclid::TypedVector2D`

    |

    = note: expected type `&euclid::TypedPoint2D<f32, euclid::UnknownUnit>`

               found type `&euclid::TypedVector2D<f32, euclid::UnknownUnit>`

    = help: here are some functions which might fulfill your needs:

            - .length()

            - .square_length()

error[E0308]: mismatched types

   --> src/azure_hl.rs:739:71

    |

739 |                                               &AzPoint::from_point_2d(offset),

    |                                                                       ^^^^^^ expected struct `euclid::TypedPoint2D`, found struct `euclid::TypedVector2D`

    |

    = note: expected type `&euclid::TypedPoint2D<f32, euclid::UnknownUnit>`

               found type `&euclid::TypedVector2D<f32, euclid::UnknownUnit>`

    = help: here are some functions which might fulfill your needs:

            - .length()

            - .square_length()

error: aborting due to previous error

error: Could not compile `azure`.
```

**Comment by nical** [comment]
This should be in a good state now, r? @jdm

File: src/azure_hl.rs [hunk 1]
```diff
@@ -1781,6 +1793,13 @@ impl AzPoint {
             y: point.y,
         }
     }
+
+    fn from_vector_2d(point: &Vector2D<AzFloat>) -> Self {
```
Comment (nox) [inline src/azure_hl.rs]: Why aren't the `from_` methods `From` impls?

File: src/azure_hl.rs [hunk 2]
```diff
@@ -161,6 +162,17 @@ impl AsAzurePoint for Point2D<AzFloat> {
     }
 }
 
+// Azure does not have a separation between points and vectors, AzPoint is used
+// to represent both
+impl AsAzurePoint for Vector2D<AzFloat> {
+    fn as_azure_point(&self) -> AzPoint {
```
Comment (nox) [inline src/azure_hl.rs]: `#[inline]`

File: src/azure_hl.rs (hunk 1 above)
Comment (nox) [inline src/azure_hl.rs]: `#[inline]`

Comment (nical) [inline src/azure_hl.rs]: No idea, I really just followed whatever was already in place (but asked myself the same question), there's also something very similar in the servo repo in the code that interacts with azure (something that should just use From/Into).

**Pull request merged**